repos/spack_repo/builtin/packages/dd4hep/package.py

```diff
@@ -230,8 +230,6 @@ def setup_run_environment(self, env: EnvironmentModifications) -> None:
         env.set("DD4HEP", self.prefix.examples)
         env.set("DD4hep_DIR", self.prefix)
         env.set("DD4hep_ROOT", self.prefix)
-        if len(self.libs.directories) > 0:
-            env.prepend_path("LD_LIBRARY_PATH", self.libs.directories[0])
```

Comment:
Removing this is not correct. The plugin mechanism that DD4hep uses relies on `LD_LIBRARY_PATH` and unless it is being set by something else many things in DD4hep won't work.

Reply:
> Removing this is not correct. The plugin mechanism that DD4hep uses relies on `LD_LIBRARY_PATH` and unless it is being set by something else many things in DD4hep won't work.

If this is a DD4hep requirement, then let DD4hep set it. Spack tries its very best to avoid the need for LD_LIBRARY_PATH by using RPATH, and having some base-level package like ROOT set it for basically every HEP stack is really annoying.